proto/storage/service_identity.proto

```diff
@@ -35,6 +35,7 @@ enum ServiceType {
   SCANNER_V4_DB_SERVICE = 15;
   // This is used when Scanner V4 is run in combo-mode.
   SCANNER_V4_SERVICE = 16;
+  REGISTRATOR_SERVICE = 17;
```

Comment:
It seems [the word registrator can refer to either side](https://en.wiktionary.org/wiki/registrator) of the transaction.
So I guess this should be:
```suggestion
  REGISTRANT_SERVICE = 17;
```
🤔

Reply:
Oh ok, thanks.